test/system/test_api_interfaces.py

```diff
@@ -635,15 +653,16 @@ def test_update_vlan(self):
             api = dut.api('interfaces')
             instance = api.update_vlan('Vxlan1', '10', '10')
             self.assertTrue(instance)
-            self.contains('vxlan vlan add 10 vni 10', dut)
+            self.may_contain('vxlan vlan 10 vni 10', dut, first=True)
+            self.may_contain('vxlan vlan add 10 vni 10', dut, last=True)
 
     def test_remove_vlan(self):
         for dut in self.duts:
             dut.config(['no interface Vxlan1', 'interface Vxlan1'])
             api = dut.api('interfaces')
             instance = api.remove_vlan('Vxlan1', '10')
             self.assertTrue(instance)
-            self.notcontains('vxlan vlan remove 10 vni 10', dut)
+            self.notcontains('vxlan vlan remove 10 vni 10 $', dut)
```

Comment:
In a prior command usage the $ was part of a new format. Does an old vs new format not apply here?

Reply:
yes, this is a test cli sequence, which I updated with the newer syntax, however to handle it properly the `remove_vlan ()` method was updated to handle both syntaxes - the newer and the deprecated one.